packages/components/src/JSONSchemaRenderer/JSONSchemaRenderer.component.js

```diff
@@ -0,0 +1,174 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import entries from 'lodash/entries';
+
+import css from './JSONSchemaRenderer.scss';
+
+const className = 'json-schema-renderer';
+
+/**
+ * UnkownTypeException
+ *
+ * @param type - The unkown type
+ * @returns {Object} An UnkownTypeException
+ */
+function UnkownTypeException(type) {
+	this.name = 'UnkownTypeException';
+	this.message = `Unkown type: ${type}`;
+}
+
+/**
+ * InvalidSchemaException
+ *
+ * @returns {Object} An InvalidSchemaException
+ */
+function InvalidSchemaException() {
+	this.name = 'InvalidSchemaException';
+	this.message = 'Invalid Schema';
+}
+
+/**
+ * textRenderer
+ *
+ * @param key
+ * @param title
+ * @param text
+ * @returns {string} - HTML markup for a text component
+ */
+function textRenderer(key, title, text) {
```

Comment:
why no using a component here ?
Text({key, title, text}) ?
by the way with component the key is not a needed props because it's internal stuff for react.